### PR TITLE
Upgrade to 2.17

### PR DESCRIPTION
**Pull request by snewcomer**
Here is just a start to potentially working towards getting Ember into https://github.com/gothinkster/realworld

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -4,18 +4,8 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: ['eslint:recommended', 'prettier'],
-  plugins: ['prettier'],
+  extends: 'eslint:recommended',
   env: {
     browser: true
-  },
-  rules: {
```
Comment (snewcomer) [inline .eslintrc.js]: This was wreaking havoc on the test suite.  Lmk if this is important to everybody and I can add it back!

**Comment by Alonski** [comment]
I'll take a look hopefully tomorrow. Can you take a lot at the other pull requests and see if they are satisfactory? I'll merge them if so.

File: .eslintrc.js (hunk 1 above)
Comment (Alonski) [reply .eslintrc.js]: Hey, we use this to lint along with prettier.
Please re-implement this and get the test suite passing.

If you need any help you can look here:
https://prettier.io/docs/en/eslint.html

**Comment by jonnii** [comment]
@Alonski i think it's better to use the settings that come out of the box tbh.

**Comment by jonnii** [comment]
Actually on second thought I'm ambivalent about this, but it's just another thing that someone who is looking at this repo has to know and install. I personally use the same prettier settings in my own projects.

**Comment by Alonski** [comment]
@jonnii Which settings do you use? I found that for Ember lines can get a bit long and the other is just a preference.
Anyways I want Eslint to lint prettier settings as well

**Comment by snewcomer** [comment]
@Alonski my bad...fixed

**Comment by snewcomer** [comment]
@Alonski just pinging this again!  no rush though :)

**Comment by Alonski** [comment]
Hey, hope I am not being a bummer.
But 2.17.2 came out :)

Could you also update the rest of the npm packages while you are at it?
```
Using D:\Projects\Fun Stuff\ember-realworld\package.json

 tslint                          4.0.2  →    5.8.0
 typescript                      2.0.0  →    2.6.2
 ember-cli-dependency-checker   ^1.4.0  →   ^2.1.0
 eslint                        ^3.19.0  →  ^4.14.0
 eslint-config-prettier         ^1.7.0  →   ^2.9.0

The following dependencies are satisfied by their declared version range, but the installed versions are behind. You can
 install the latest versions without modifying your package file by using npm update. If you want to update the dependen
cies in your package file anyway, run ncu -a.

 broccoli-asset-rev                     ^2.5.0  →   ^2.6.0
 ember-cli                             ^2.17.0  →  ^2.17.2
 ember-cli-babel                        ^6.6.0  →  ^6.11.0
 ember-cli-eslint                       ^4.2.1  →   ^4.2.3
 ember-cli-htmlbars                     ^2.0.1  →   ^2.0.3
 ember-cli-htmlbars-inline-precompile   ^1.0.0  →   ^1.0.2
 ember-cli-inject-live-reload           ^1.6.1  →   ^1.7.0
 ember-cli-moment-shim                  ^3.1.0  →   ^3.5.0
 ember-cli-qunit                        ^4.1.1  →   ^4.2.1
 ember-data                            ^2.17.0  →  ^2.18.0
 ember-moment                           ^7.3.0  →   ^7.5.0
 ember-resolver                         ^4.1.0  →   ^4.5.0
 ember-route-action-helper              ^2.0.3  →   ^2.0.6
 ember-simple-auth                      ^1.4.1  →   ^1.4.2
 eslint-plugin-prettier                 ^2.0.1  →   ^2.4.0
 loader.js                              ^4.3.0  →   ^4.6.0
 prettier                               ^1.2.2  →   ^1.9.2

Run ncu with -u to upgrade package.json
```
You can use the npm package: npm-check-updates to get this list

**Comment by Alonski** [comment]
Hey man
Are you planning to update any more or should I merge this in?

**Comment by Alonski** [comment]
Closing this because of #33

**Pull request closed without merging**